src/classes/STG_PanelPaymentMapping_CTRL.cls

```diff
@@ -91,59 +92,58 @@ public with sharing class STG_PanelPaymentMapping_CTRL extends STG_Panel {
     }
 
     /*********************************************************************************************************
-    * @description Initializes the targetfields SelectOption list for the selected object
+    * @description Initializes the targetFields SelectOption list for the selected object
+    * @return List of SelectOption with available fields for mapping.
     */
     public list<SelectOption> getTargetFields(){
 
-        targetfields.clear();
-        set<string> knownfields;
-        map<string, Schema.Sobjectfield> targetTokenMap = new map<string, Schema.Sobjectfield>();
-
-        targetTokenMap = Schema.SObjectType.npe01__OppPayment__c.fields.getMap().clone();
-        knownfields = new set<string>{
+        targetFields.clear();
+        Set<String> knownPaymentFieldsToExclude = new Set<String>{
             'npe01__Opportunity__c', 'npe01__Paid__c', 'npe01__Payment_Amount__c', 'npe01__Payment_Date__c',
             'npe01__Written_Off__c'};
+        Map<String, Schema.Sobjectfield> targetTokenMap = new Map<String, Schema.Sobjectfield>();
 
+        targetTokenMap = Schema.SObjectType.npe01__OppPayment__c.fields.getMap().clone();
+       
         //pop out the known fields
-        for (string s : knownfields){
+        for (String s : knownPaymentFieldsToExclude){
```

Comment:
Let's create a descriptive var name to indicate what the element is, ie let's use `fieldName` instead of the `s`.